src/controller/CHIPDeviceControllerFactory.cpp

```diff
@@ -175,6 +179,7 @@ void DeviceControllerFactory::PopulateInitParams(ControllerInitParams & controll
 CHIP_ERROR DeviceControllerFactory::SetupController(SetupParams params, DeviceController & controller)
 {
     VerifyOrReturnError(mSystemState != nullptr, CHIP_ERROR_INCORRECT_STATE);
+    mStorage = params.storageDelegate;
```

Comment:
The storage in SetupParams is the fabric-specific storage for this specific controller.  mStorage is our cutting-across-all-fabrics storage.  We really shouldn't use the former for the latter.

I suspect taking this line out is fine; we should have had Init() called on us with the right delegate passed in.

Reply:
Done